assets/app/views/project.html

```diff
@@ -131,10 +131,11 @@ <h2 class="service" ng-if="!displayRouteByService[serviceId]">
           <div ng-repeat="deployment in deployments | orderObjectsByDate : true track by (deployment | uid)" 
                ng-if="(podsByDeployment[deployment.metadata.name] | hashSize) > 0">
 
+            <!-- Make sure deploymentConfigs are loaded before determining if the deployment config is missing. -->
             <overview-deployment
               rc="deployment"
               deployment-config-id="deploymentConfigId"
-              deployment-config="deploymentConfigs[deploymentConfigId]"
+              deployment-config-missing="deploymentConfigs && deploymentConfigs[deploymentConfigId] == null"
```

Comment:
The `deploymentConfig == null` check was previously in _overview-deployment.html. I wasn't sure if it was that way for a reason.